corehq/apps/users/views/mobile/custom_data_fields.py

```diff
@@ -68,6 +68,13 @@ def dispatch(self, request, *args, **kwargs):
     def update_existing_models(self):
         remove_unused_custom_fields_from_users_task.delay(self.domain)
 
+    def update_profile_required(self, profile_required_for_user_type):
```

Comment:
I think this needs to also be implemented by other subclasses of `CustomDataModelMixin`